python-bindings/chiapos.cpp

```diff
@@ -80,20 +80,14 @@ PYBIND11_MODULE(chiapos, m)
         .def(
             "get_memo",
             [](DiskProver &dp) {
-                uint8_t *memo = new uint8_t[dp.GetMemoSize()];
-                dp.GetMemo(memo);
-                py::bytes ret = py::bytes(reinterpret_cast<char *>(memo), dp.GetMemoSize());
-                delete[] memo;
-                return ret;
+                std::vector<uint8_t> memo = dp.GetMemo();
+                return py::bytes(reinterpret_cast<char*>(memo.data()), memo.size());
```

Comment:
Its probably not a big deal, but this will copy the buffer twice, where i would think once is enough

Reply:
Changed!